### PR TITLE
added icd 9 cm

### PR DESCRIPTION
**Pull request by thutuva**
Added icd 9 cm to csv file, to sidebar.js, and created a page for it that does not display the data dictionary and says it's too large.

I also changed all references in CSV file to 'CMS' to 'Centers for Medicare & Medicaid Services (CMS)'

### CODE REVIEW

File: static/data/terminology.csv [hunk 1]
```diff
@@ -3,16 +3,17 @@ Administrative Codes,Admit Source,National Uniform Billing Committee,,,"/termino
...
 Administrative Codes,Place of Service,Centers for Medicare & Medicaid Services (CMS),,,"/terminology/administrative-codes/place-of-service"
 Administrative Codes,Present on Admission,Centers for Medicare & Medicaid Services (CMS),,,"/terminology/administrative-codes/present-on-admission"
 Administrative Codes,Revenue Center,National Uniform Billing Committee,,,"/terminology/administrative-codes/revenue-center"
 Conditions and Procedures,APR-DRG,3M,,,/terminology/conditions-and-procedures/apr-drg
 Conditions and Procedures,HCPCS Level II,Centers for Medicare & Medicaid Services (CMS),,,/terminology/conditions-and-procedures/hcpcs-level-2
 Conditions and Procedures,ICD-10-CM,Centers for Disease Control and Prevention (CDC),,,/terminology/conditions-and-procedures/icd-10-cm
 Conditions and Procedures,ICD-10-PCS,Centers for Medicare & Medicaid Services (CMS),,,/terminology/conditions-and-procedures/icd-10-pcs
-Conditions and Procedures,MDC,CMS,,,/terminology/conditions-and-procedures/mdc
+Conditions and Procedures,ICD-9-CM,Centers for Medicare & Medicaid Services (CMS),10.01.2014, 05.10.2023,/terminology/conditions-and-procedures/icd-10-pcs
```
Comment (aneiderhiser) [inline static/data/terminology.csv:15]: this link goes to the icd-10-pcs page -- it needs to be changed to go to the icd-9-cm page

Comment (aneiderhiser) [reply static/data/terminology.csv:15]: literally updating this /terminology/conditions-and-procedures/icd-10-pcs to /terminology/conditions-and-procedures/icd-9-cm should work i think

**Review by aneiderhiser — CHANGES_REQUESTED** [review]
I reviewed including docs and test running the sql from quickstart in snowflake.  Everything looks good except for the one minor link issue i commented on.

File: static/data/terminology.csv (hunk 1 above)
Comment (thutuva) [reply static/data/terminology.csv]: nice catch!  The text has been corrected to icd-9-cm.

**Pull request merged**